model/task/task.go

```diff
@@ -463,6 +469,18 @@ func (tr TestResult) GetLogURL(viewer evergreen.LogViewer) string {
 			url.QueryEscape(tr.GroupID),
 			tr.LineNum,
 		)
+	case evergreen.LogViewerParsley:
+		if parsleyURL == "" {
+			return ""
+		}
+		for _, url := range deprecatedLobsterURLs {
+			if strings.Contains(tr.URL, url) {
+				updatedResmokeParsleyURL := strings.Replace(tr.URL, fmt.Sprintf("%s/build", url), parsleyURL+"/resmoke", 1)
```

Comment:
Would it make more sense to link to Parsley from Logkeeper instead, in the same way that we currently redirect to Lobster? I think it would avoid the URL surgery here and simplify the code.

Reply:
Yes, i think so! But we would still need to keep this URL surgery, for now, to account for older links.